docs/central.md

```diff
@@ -242,6 +241,9 @@ user home or to use environment variables for publishing from CI servers.
     ORG_GRADLE_PROJECT_signingInMemoryKeyPassword=some_password
     ```
 
+Note that the username/password here is *not* the same one you use to login; Sonatype publishing
+requires a username/password that was [generated via user tokens](https://central.sonatype.org/publish/generate-portal-token/).
```

Comment:
Just a small change to clarify that Central Portal user tokens are separate from user tokens for the old sites:
```suggestion
requires a username/password that was generated via user tokens. The user token needs to be obtained on [Sonatype OSS](https://central.sonatype.org/publish/generate-token/) or the [Central Portal](https://central.sonatype.org/publish/generate-portal-token/) depending on where you publish.
```